src/components/forms/text-area/text-area.tsx

```diff
@@ -15,6 +15,11 @@ const StyledTextArea = styled.textarea<{ hasError?: boolean }>`
 
   background: ${theme.palette.background.leadbox};
   border: 2px solid ${theme.palette.background.leadbox};
+
+  &:hover {
+    border: 1px solid ${theme.palette.text.default};
+  }
```

Comment:
This is a design change. Can you help me where this is coming from? I remember something mentioned by eric before. What's the purpose of this if the highlight is not maintained. There is not a single highlight for when I'm inside a field. Is this an error or by design? Because I would reject that design then, it's not accessible. Can you help me?

Reply:
This a design change, which was mentioned I think by Eric. The other form components act like this too

Reply:
Ok. We leave it as is. But we have to come back to this because it's not good for the user.